api/src/hyp3_api/handlers.py

```diff
@@ -5,16 +5,19 @@
 
 
 def submit_job(body):
+    parameters = body['parameters']
     job = BATCH_CLIENT.submit_job(
-        jobName=body['granule'],
+        jobName=parameters['granule'],
         jobQueue=environ['JOB_QUEUE'],
         jobDefinition=environ['JOB_DEFINITION'],
-        parameters=body
+        parameters=parameters
     )
+    print(job)
     response = {
         'jobId': job['jobId'],
-        'parameters': body,
+        'parameters': parameters,
     }
```

Comment:
`response` could include everything they gave us, plus the job id?  so `body[jobId] = job['jobId']`; return job`?